app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -1,11 +1,27 @@
 import React from "react";
 
+import * as yup from "yup";
+
 import ContactActions from "./Contact/ContactActions";
 import ContactAvatar from "./Contact/ContactAvatar";
 import ContactDetailText from "./Contact/ContactDetailText";
 
 import { NAMES } from "../Notes/constants";
 
+export const CONTACTS_FORM_INITIAL_FORM_VALUES = {
+  firstName: "",
+  lastName: "",
+  email: "",
+  role: "",
+};
+
+export const CONTACTS_FORM_VALIDATION_SCHEMA = yup.object().shape({
+  firstName: yup.string().required("First Name is required"),
+  lastName: yup.string().required("Last Name is required"),
+  email: yup.string().required("Email Address is required"),
+  role: yup.object().required("Role is required").nullable(),
```

Comment:
This is kind of confusing to me. The `role` is `required` as well as it's `nullable`? Can you please point out the business aspect of this logic?